api/app/src/mappings/garmin/activity-types.ts

```diff
@@ -1,6 +1,23 @@
 import { capitalize } from "lodash";
-import z from "zod";
+import { z } from "zod";
 
+function containsStringIndex(map: Record<ActivityType, string>, index: string): boolean {
+  return Object.prototype.hasOwnProperty.call(map, index);
+}
+
+export function activityTypeReadable(activityType: string): string {
+  // will the activity type mapping if it exists, and otherwise for example will change strings from:
+  //    - "SWIM_TO_BIKE_TRANSITION_V2" to "Swim To Bike Transition"
+  //    - "CROSS_COUNTRY_SKIING_WS" to "Cross Country Skiing"
+  const stringToUse = containsStringIndex(activityTypeMapping, activityType)
+    ? activityTypeMapping[activityType as ActivityType]
+    : activityType.replace(/_|V\d+|WS/g, " ");
+  return capitalize(stringToUse).trim();
+}
+
+// see https://github.com/metriport/metriport-internal/issues/865 for context...
+// these enums are current as of the Garmin API @ 1.0.3, and are out-of-date at the time of writing
+// this @ 1.0.5 - keeping for backwards compatibility
 export const activityTypeSchema = z.enum([
```

Comment:
Where is this used now?

Reply:
just to make the ActivityType enum https://github.com/metriport/metriport/blob/a4bb37cf978ff1b8848c6b96d3ce8fdd73897cf1/api/app/src/mappings/garmin/activity-types.ts#L123

Reply:
could just change this to a <string, string> map though